src/main/java/com/exadel/discountwebapp/location/controller/LocationController.java

```diff
@@ -44,15 +43,15 @@ public LocationResponseVO getLocationById(@PathVariable Long id) {
 
     // TODO: tests for this controller
     @GetMapping("/cities")
-    public List<CityResponseVO> getAllCitiesByCountry(
```

Comment:
it seems like will break the UI after this change

Reply:
We just have called concerning it and they say it's okay, they will modify on front to process List of Strings